app.js

```diff
@@ -8,9 +8,11 @@ const rateLimit = require("express-rate-limit");
 const mongoSanitize = require("express-mongo-sanitize");
 const key = require("./utils/libs/gen-key");
 
+const authRouter = require('./routes/user');
 const AppError = require("./utils/libs/appError");
 const globalErrorHandler = require("./controllers/errorController");
 const { successResMsg } = require("./utils/libs/response");
+const bodyParser = require("body-parser");
 
```

Comment:
why are you using body-parser package here, express package already provide the same functionalities.

Reply:
Yea i know, it was a mistake. Remember i told you my body obj wasn't parsing for some reason so i installed the body parser to test i am no longer using It I'll remove it

Reply:
Also you are having conflicts in your package.json, please can you rectify that, I can't merge.

Reply:
> Yea i know, it was a mistake. Remember i told you my body obj wasn't parsing for some reason so i installed the body parser to test i am no longer using It I'll remove it

Ok